frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -56,7 +58,7 @@ export const saveParcelDataLayerResponse = (
       }),
     );
   } else {
-    toast.warning(`Failed to find PID/PIN. Ensure that the searched PID/PIN is valid`);
+    toast.warning(`Failed to find parcel layer data. Ensure that the search criteria is valid`);
```

Comment:
This is used for pid/pin and also lat/lng searches.

Reply:
I noticed this when searching for an address but the PID/PIN warning popped up, thanks for clarifying!